src/auth.js

```diff
@@ -21,11 +22,13 @@ router.post ('/login', middleware.isNotAuthenticated, (req, res, next) => {
     return;
   }
 
-  models.User.find({where : {
-    email, password
-  }}).then(user => {
-    if (user) sendUserData(user, res);
-    else res.sendStatus(401);
+  models.User.find({where : { email }}).then(user => {
+    if (user) {
+      bcrypt.compare(password, user.password, function(err, response) {
+        if (response == true) sendUserData(user, res);
```

Comment:
Just add `if (err) return next (err);` Basically this will pass the control and the error to the error handling middleware which will send a 500 code to the client and log the error